sparse/_coo/common.py

```diff
@@ -64,7 +64,7 @@ def linear_loc(coords, shape):
         return np.ravel_multi_index(coords, shape)
 
 
-def tensordot(a, b, axes=2):
+def tensordot(a, b, axes=2, return_type=None):
```

Comment:
```suggestion
def tensordot(a, b, axes=2, *, return_type=None):
```

Reply:
This will make it so that if NumPy adds any additional `kwargs`, we remain compatible in the future.